OwnTracks-Presence/owntracks-presence-app.groovy

```diff
@@ -119,37 +119,40 @@ def update (devices) {
     data = parseJson(request.body)
     ifDebug("DATA: ${data} PARAMS: ${params}")
     if (data._type == "transition") {
-          // https://owntracks.org/booklet/tech/json/#_typetransition
-          ifDebug("Received transition event")
-   	      def event = data.event
-   	      def user = params.user
-          def location = data.desc  
-          def deviceName = location + "-" + user
-          def device = devices.find { it.currentValue("region") + "-" + it.currentValue("user") == deviceName }
-   	      ifDebug("event: ${event} device: ${device} location: ${location} user: ${user} deviceName: ${deviceName}")     
- 	      if (location){
+	    // https://owntracks.org/booklet/tech/json/#_typetransition
+	    ifDebug("Received transition event")
+	    def event = data.event
+	    def user = params.user
+	    def location = data.desc 
+	    def deviceName = location + "-" + user
+	    def device = devices.find { it.currentValue("region") + "-" + it.currentValue("user") == deviceName }
+	    if (!device) {
```

Comment:
Does this work as a boolean?

Reply:
I see what you are trying to do by giving people more options, but I have never seen any driver or app store usable data inside the name of the device. So I am unsure why this option needs to be there other than for backwards compatibility reasons, but honestly having consistent usage between apps/drivers and less code bloat is more important than backwards compatibility and will make modifying down the line easier.

Reply:
You’re absolutely right. Then there are no “either here or there” situations.  It also simplifies instructions.